tests/tests-node/esmock.node.test.js

```diff
@@ -434,6 +434,18 @@ test('should throw error when strict mock definition not found', async () => {
   assert.deepEqual(pathWrapPartial.basename('/dog.png'), 'dog.png')
 })
 
+test('should error when "strictest" called with defs: {}', async () => {
+  await assert.rejects(async () => esmock.strictest(
+    '../local/strictest.js', {}
```

Comment:
I propose to use importsCoreLocalAndPackage.js instead of my simple demo - strictest.js